pkg/deployment/deployment.go

```diff
@@ -315,6 +315,32 @@ func (builder *Builder) WithServiceAccountName(serviceAccountName string) *Build
 	return builder
 }
 
+// WithVolumes sets list of volumes that can be mounted by containers belonging to the deployment.
+func (builder *Builder) WithVolumes(deployVolumes []coreV1.Volume) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Adding %d volumes to deployment %s in namespace %s",
```

Comment:
Looks like this glog entry should be moved to after the  len(deployVolumes) == 0 check

Reply:
yeah, I was thinking about it as well but it looks harmless  so left it here. please let me know if it should be moved further, thanks

Reply:
I would.
glog.V(100).Infof("The volume list of the deployment is empty") would come in case of 0

and "Adding %d volumes to deployment %s in namespace %s" in case > 0

Reply:
thanks, updated